gem_rake_helper.rb

```diff
@@ -21,7 +21,7 @@ def release_gem(*args)
   exempt_tags << '--tags ~@encoding' unless Object.const_defined?(:Encoding)
   exempt_tags << '--tags ~@nowindows' if Gem.win_platform?
   exempt_tags << '--tags ~@travishatesme' if ENV['TRAVIS'] == 'true'
-  t.cucumber_opts = "--color #{exempt_tags.join(' ')} --strict --format #{ENV['CUCUMBER_FORMAT'] || 'Fivemat'}"
+  t.cucumber_opts = "--require features --color #{exempt_tags.join(' ')} --strict --format #{ENV['CUCUMBER_FORMAT'] || 'Fivemat'}"
 end
```

Comment:
This `--require` is necessary otherwise cucumber does not work with `features/cli`-directory. You may want to move some of the `cli_*`-files there too.